api/client/cli.go

```diff
@@ -123,6 +123,18 @@ func (cli *DockerCli) ImagesFormat() string {
 	return cli.configFile.ImagesFormat
 }
 
+// NetworksFormat returns the format string specified in the configuration.
+// String contains columns and format specification, for example {{ID}}\t{{Name}}
+func (cli *DockerCli) NetworksFormat() string {
+	return cli.configFile.NetworksFormat
+}
```

Comment:
Hum true :sweat_smile: I just mimic the current behavior but I could rework them all to use `dockerCli.ConfigFile().NetworksFormat` I think :wink: